ff-netF48-server-sdk.csproj

```diff
@@ -8,9 +8,9 @@
         <PackageId>ff-dotnet-server-sdk</PackageId>
         <RootNamespace>io.harness.cfsdk</RootNamespace>
         <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
-        <Version>1.7.0</Version>
+        <Version>1.7.0-rc1</Version>
```

Comment:
FYI in general `-rc.1` is favored, as it separates the tokens of `-rc` and the number; in semver 2, the numbers get treated as numbers if separated. (so rc2 does not come after rc11)

Reply:
Thanks @bartelink.